### PR TITLE
Fix

### PR DESCRIPTION
**Pull request by AlekseySayarkin**

### CODE REVIEW

File: service/src/main/java/com/epam/esm/service/CertificateRequestBody.java [hunk 1]
```diff
@@ -3,15 +3,17 @@
 public class CertificateRequestBody {
 
     private String content;
-    private String sort;
+    private String sortType;
```
Comment (churaalena) [inline service/src/main/java/com/epam/esm/service/CertificateRequestBody.java:6]: You can use enum here

File: service/src/main/java/com/epam/esm/service/util/CertificateValidator.java [hunk 2]
```diff
@@ -12,44 +12,66 @@ private CertificateValidator() {
...
+        if (updateDate.isBefore(createDate)) {
+            throw new ServiceException("Failed to validate: certificate update date is before create date",
+                    ErrorCodeEnum.CERTIFICATE_VALIDATION_ERROR);
+        }
     }
 
-    private static boolean isValidDuration(int duration) {
-        return duration > 0;
+    private static void validateDuration(int duration) throws ServiceException {
+        if (duration < 0) {
```
Comment (churaalena) [inline service/src/main/java/com/epam/esm/service/util/CertificateValidator.java:72]: Less or equal 0

**Review by churaalena — APPROVED** [review]

File: service/src/main/java/com/epam/esm/service/CertificateRequestBody.java (hunk 1 above)
Comment (AlekseySayarkin) [reply service/src/main/java/com/epam/esm/service/CertificateRequestBody.java]: done

File: service/src/main/java/com/epam/esm/service/util/CertificateValidator.java (hunk 2 above)
Comment (AlekseySayarkin) [reply service/src/main/java/com/epam/esm/service/util/CertificateValidator.java]: done

**Pull request merged**